typescript/infra/config/environments/mainnet2/agent.ts

```diff
@@ -1,23 +1,30 @@
-import { ALL_KEY_ROLES } from '../../../src/agents/roles';
+import { ALL_KEY_ROLES, KEY_ROLE_ENUM } from '../../../src/agents/roles';
 import { AgentConfig } from '../../../src/config';
 import {
   ConnectionType,
   GasPaymentEnforcementPolicyType,
 } from '../../../src/config/agent';
 import { Contexts } from '../../contexts';
+import { helloworldMatchingList } from '../../utils';
+import { helloWorld } from '../mainnet2/helloworld';
 
 import { MainnetChains, chainNames, environment } from './chains';
 // import { helloWorld } from './helloworld';
```

Comment:
nit: should we use this import instead of the one on line 9? the one on line 9's `../mainnet2` seems redundant

Reply:
Yeah, IDK what the IDE was thinking. Fixed.